DESCRIPTION

```diff
@@ -1,16 +1,16 @@
-Package: cellengine
-Type: Package
-Title: CellEngine API Toolkit
-Version: 0.1.0
-Authors@R: c(
-    person("Zach", "Bjornson", email = "zbjornson@primitybio.com",
-           role = c("aut", "cre")))
-Description: Provides an easy interface for manipulating experiments, FCS files,
-    gates and populations, and exporting data from CellEngine.
-License: Apache License (== 2.0)
-Encoding: UTF-8
-LazyData: true
-Depends: R (>= 3.0.0)
-Imports: httr (>= 1.3.0), jsonlite
-RoxygenNote: 6.0.1
-Suggests: httptest
+Package: cellengine
+Type: Package
+Title: CellEngine API Toolkit
+Version: 0.1.0
+Authors@R: c(
+    person("Zach", "Bjornson", email = "zbjornson@primitybio.com",
+           role = c("aut", "cre")))
+Description: Provides an easy interface for manipulating experiments, FCS files,
+    gates and populations, and exporting data from CellEngine.
+License: Apache License (== 2.0)
+Encoding: UTF-8
+LazyData: true
+Depends: R (>= 3.0.0)
+Imports: httptest(>= 3.2.2)
```

Comment:
👍 for adding the httptest version. Hadley says to keep things like testing tools in `Suggests` though, see http://r-pkgs.had.co.nz/description.html#dependencies

It looks like `jsonlite` and `httr` got dropped?

Reply:
Fixed